src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -57,6 +65,16 @@ class Filter extends Component<Props> {
     return !this.existy.hasArtworks
   }
 
+  hideMobileActionSheet = () => {
+    this.setState({ showMobileActionSheet: false })
+    document.body.style.overflowY = "auto"
```

Comment:
As an aside, this was sort of a get it out the door hack; I'm sure there are better ways of handling this if need be, via something like react-side-effect. Could also be fine for what it is!